spec/README.md

```diff
@@ -79,18 +79,18 @@ a version 1 app can verify version 1, 2, 3, or 4 proofs. However, a version
 <table>
   <thead>
     <tr>
-      <th rowspan=2>Version</th>
+      <th rowspan=2>App Identity<br />Version</th>
       <th rowspan=2>Nonce</th>
       <th rowspan=2>Digest Algorithm</th>
-      <th colspan=4>Can Verify</th>
+      <th colspan=4>Can Verify Proof From Version</th>
     </tr>
     <tr><th>1</th><th>2</th><th>3</th><th>4</th></tr>
   </thead>
   <tbody>
-    <tr><th>1</th><td>random</td><td>SHA 256</td><td>✅</td><td>✅</td><td>✅</td><td>✅</td></tr>
```

Comment:
Mentioned in Slack, but `☐` and `☑` seem like they’d look a bit grey and similar. Should we consider using `➖` and `➕` or something similar? No problem using less-colourful markers.

Alternatively, we could just use `Yes` and `No`.

As a side note, and this is something that I can do mechanically before release, I think we want to explicitly refer to the algorithms as `SHA2-256` et al. as when SHA3 support is readily available for our supported languages, we would want to add that and those are `SHA3-224`, `SHA3-256`, `SHA3-384`, `SHA3-512`, `SHAKE128`, and `SHAKE256` (I’m not sure which of these algorithms we will support; more research is required).